exporter/kafkaexporter/config.modified.go

```diff
@@ -53,6 +53,9 @@ type Config struct {
 
 	// Compression defines the compression method and compression level, if applicable.
 	Compression Compression `mapstructure:"compression"`
+
+	// Debug. Log info for spans that exceed Producer.MaxMessageBytes early.
+	Debug bool `mapstructure:"debug"`
```

Comment:
This looks very specific to an implementation, couldn't we just add a log warning instead of this very specific option?

Reply:
I wanted the option to quickly be able to turn it off and on in case it pukes.